### PR TITLE
[nginx-ingress-controller]: Add support for geoip in stats

### LINKED ISSUE #1101: Nginx Ingress Controller - Support for geoip
Hi @aledbf ,

is it possibile to use directives

`geoip_country /usr/share/GeoIP/GeoIP.dat;
vhost_traffic_status_filter_by_set_key $geoip_country_code country::*;`

in the latest version of Nginx ingress controller?

I need it to to calculate traffic for individual country.

Thank you.

### PR DESCRIPTION
**Pull request by aledbf**
fixes #1101

### CODE REVIEW

File: ingress/controllers/nginx/nginx.tmpl [hunk 1]
```diff
@@ -16,7 +16,13 @@ events {
 }
 
 http {
-    {{ if $cfg.enableVtsStatus}}vhost_traffic_status_zone shared:vhost_traffic_status:{{ $cfg.vtsStatusZoneSize }};{{ end }}
+    geoip_country       /etc/nginx/GeoIP.dat;
```
Comment (bprashanth) [inline ingress/controllers/nginx/nginx.tmpl]: please update the REAMDE with an example of who this is useful, or some eager newbie will think it's a dead module and optimize it out of the build for sure

**Comment by bprashanth** [comment]
LGTM but for the one comment

File: ingress/controllers/nginx/nginx.tmpl (hunk 1 above)
Comment (aledbf) [inline ingress/controllers/nginx/nginx.tmpl]: done

**Comment by Karthickarumugam** [comment]
How to get country code in log_format?

**Pull request merged**